src/lib/y2partitioner/widgets/format_and_mount.rb

```diff
@@ -250,26 +271,29 @@ def handle(event)
     private
 
       def mount_device
-        @controller.mount_point = @mount_point_widget.value.to_s
-        @fstab_options_widget.enable
         @mount_point_widget.enable
+        mount_point_change
       end
 
       def dont_mount_device
-        @controller.mount_point = ""
+        @controller.remove_mount_point
         @fstab_options_widget.disable
         @mount_point_widget.disable
       end
 
       def mount_point_change
-        mount_point = @mount_point_widget.value.to_s
-        @controller.mount_point = mount_point
-        if mount_point.nil? || mount_point.empty?
+        if mount_path.empty?
```

Comment:
Is this safe since at other places we also check for `nil?`  ?

Reply:
In this case, `mount_path` is a method belonging to the widget. It simply returns the string given by the user for the mount point, so we only check that the string is not empty.